### PR TITLE
Arcade Loot Improvements + Battlemachine Tweaks

### PR DESCRIPTION
**Pull request by BurgerLUA**
[Changelogs]:
:cl: BurgerBB
add: Adds several new toy loot to the arcade machine.
balance: Rebalanced the arcade machine loot. Battlemachines now have a 0.5 second delay instead of a second delay between actions.
fix: Fixed a bug that would not allow the one in a million pulse rifle to spawn.
/:cl:

[why]: Arcade loot is shit, especially with a bunch of clutter cluttering it. Loot is reweighted and reedited to make it so that generally useful things have a higher chance to vend, while useless ones don't.

### CODE REVIEW

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -6,55 +11,67 @@
...
+		/obj/item/grenade/chem_grenade/glitter/white = ARCADE_WEIGHT_TRICK,
+
+		/obj/item/extendohand/acme = ARCADE_WEIGHT_TRICK,
+		/obj/item/card/emagfake	= ARCADE_WEIGHT_TRICK,
+		/obj/item/clothing/shoes/wheelys = ARCADE_WEIGHT_RARE,
+		/obj/item/clothing/shoes/kindleKicks = ARCADE_WEIGHT_RARE,
+		/obj/item/storage/belt/military/snack = ARCADE_WEIGHT_RARE,
+
+		/obj/item/clothing/mask/fakemoustache/italian = ARCADE_WEIGHT_RARE,
+		/obj/item/disk/nuclear/fake = ARCADE_WEIGHT_RARE
```
Comment (Trilbyspaceclone) [inline code/game/machinery/computer/arcade.dm]: Can unlock illegal tech node

File: code/game/machinery/computer/arcade.dm [hunk 2]
```diff
@@ -6,55 +11,67 @@
...
+		/obj/item/toy/talking/griffin = ARCADE_WEIGHT_USELESS,
+		/obj/item/toy/talking/owl = ARCADE_WEIGHT_USELESS,
+		/obj/item/toy/toy_dagger = ARCADE_WEIGHT_TRICK,
+		/obj/item/toy/toy_xeno = ARCADE_WEIGHT_TRICK,
+		/obj/item/toy/windupToolbox = ARCADE_WEIGHT_TRICK,
+
+		/obj/item/twohanded/dualsaber/toy = ARCADE_WEIGHT_RARE,
+		/mob/living/simple_animal/bot/secbot/grievous/toy = ARCADE_WEIGHT_RARE,
+		/obj/item/clothing/mask/facehugger/toy = ARCADE_WEIGHT_RARE,
+		/obj/item/gun/ballistic/automatic/toy/pistol/unrestricted = ARCADE_WEIGHT_TRICK,
```
Comment (Trilbyspaceclone) [inline code/game/machinery/computer/arcade.dm:46]: Can unlock illegal tech

Comment (BurgerLUA) [reply code/game/machinery/computer/arcade.dm:46]: Which one?

Comment (BurgerLUA) [reply code/game/machinery/computer/arcade.dm:46]: aaaaaaaaaaa which one?

**Review by silicons — APPROVED** [review]
honestly why the crap does Topic() ever fucking sleep instead of just having a cooldown check at the top jesus christ arcade code why.

**Pull request merged**